packages/core/test/financial-templates/perpetual-multiparty/PerpetualLiquidatable.js

```diff
@@ -96,10 +96,11 @@ contract("PerpetualLiquidatable", function(accounts) {
     timer = await Timer.deployed();
     await timer.setCurrentTime(startTime);
 
-    const tokenFactory = await TokenFactory.deployed();
-
     // Create Collateral and Synthetic ERC20's
     collateralToken = await Token.new("UMA", "UMA", 18, { from: contractDeployer });
+    syntheticToken = await SyntheticTokenExclusiveMinter.new("Test UMA Token", "UMAETH", 18, {
```

Comment:
Unrelated to this PR but I feel we should be more consistent through the repo with how we name tokens in tests. [here](https://github.com/UMAprotocol/protocol/blob/b3812033c066584fb42b37303b3d10491df7a71e/packages/core/test/financial-templates/perpetual-multiparty/PerpetualLiquidatable.js#L142) we create `"Test UMA Token", "ETHBTC"` but [here](https://github.com/UMAprotocol/protocol/blob/master/packages/core/test/financial-templates/perpetual-multiparty/PerpetualLiquidatable.js#L142) we create `"Test UMA Token","UMAETH"`. I've seen other examples as well.

Reply:
I created an issue to track this: https://github.com/UMAprotocol/protocol/issues/2137

Reply:
Yeah.... good idea.